api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -0,0 +1,120 @@
+package v1alpha1
+
+import (
+	"github.com/openshift-kni/performance-addon-operators/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this PerformanceProfile to the Hub version (v1).
+func (curr *PerformanceProfile) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1.PerformanceProfile)
+
+	// ObjectMeta
+	dst.ObjectMeta = curr.ObjectMeta
+
+	// Spec
+	dst.Spec.CPU = new(v1.CPU)
+	hugePagesSize := v1.HugePageSize(*curr.Spec.HugePages.DefaultHugePagesSize)
+	reserved := v1.CPUSet(*curr.Spec.CPU.Reserved)
+	dst.Spec.CPU.Reserved = &reserved
+	isolated := v1.CPUSet(*curr.Spec.CPU.Isolated)
+	dst.Spec.CPU.Isolated = &isolated
+	dst.Spec.CPU.BalanceIsolated = curr.Spec.CPU.BalanceIsolated
```

Comment:
hm you assign pointest here, it hard to say if it will affect the behavior, but better to do
```
dst.Spec.CPU.BalanceIsolated = pointer.BoolPtr(*curr.Spec.CPU.BalanceIsolated)
```
and for sure you should check that `curr.Spec.CPU.BalanceIsolated != nil`

Reply:
ok

Reply:
done